### DISCUSSION TITLE
incomplete-text

### DISCUSSION THREAD

**Discussion by giscus** [listenzz/RNDemo#12] [Announcements]
# incomplete-text

React Native 开发指南 | 教程

https://todoit.tech/incomplete-text.html

**Comment by hdjhdjhdj** [comment]
text-polyfill.ts 这个ts要在哪里引入比较好

**Reply by listenzz** [reply]
应用启动入口，index 文件

**Reply by hdjhdjhdj** [reply]
index.js文件。这样引入吗？
import './src/plugins/textPolyfill';
import './src/plugins/shadowPolyfill';

**Reply by hdjhdjhdj** [reply]
感谢大佬，有效果了

**Comment by hdjhdjhdj** [comment]
圆角的card加阴影
边缘会这样，有什么办法解决吗？
![Uploading 微信图片_20241016175159.jpg…]()

**Reply by hdjhdjhdj** [reply]
https://github.com/user-attachments/assets/13dd1c94-7d10-4b0c-b76e-327f3f7f9768

**Comment by listenzz** [comment]
样式配置发出来看看

**Reply by hdjhdjhdj** [reply]
我找到问题了。是布局有问题
我的card是用在 FlatList.renderItem
FlatList设置了padding和overflow: 'hidden'就会这样了
